Homework 2/SalesRepresentativeOrder/src/main/java/ro/siit/BubbleSort.java

```diff
@@ -0,0 +1,30 @@
+package ro.siit;
+
+import static ro.siit.SalesRepresentativeOrder.salesRep;
+
+/**
+ * This is the bubblesort() algorithm.
+ * It sorts the values of the SaleRepresentative revenues, and then it also sorts the names of the SalesRepresentatives according to their revenue.
+ */
+
+public class BubbleSort {
+    public static void bubbleSort() {
+        int a, b, t;
+        String t2;
+
+
+        for (a = 1; a < salesRep.length; a++) {
```

Comment:
This is not a Bubble Sort implementation. This is a brute fore sort.
Bubble sort has a stop condition. Please investigate it.